src/rard/research/views/concordance.py

```diff
@@ -1,41 +1,79 @@
+from django.core import paginator
+from django.core.paginator import Paginator
 from django.contrib.auth.mixins import (LoginRequiredMixin,
                                         PermissionRequiredMixin)
+from django.contrib.auth.decorators import login_required, permission_required
 from django.http import Http404
-from django.shortcuts import get_object_or_404
+from django.shortcuts import get_object_or_404, render
 from django.utils.decorators import method_decorator
 from django.views.decorators.http import require_POST
-from django.views.generic import ListView
+from django.views.generic import View
 from django.views.generic.edit import CreateView, DeleteView, UpdateView
 
 from rard.research.models import (AnonymousFragment, Concordance, Fragment,
-                                  OriginalText)
+                                  OriginalText, original_text)
```

Comment:
That's the other one I had spotted that I couldn't find yesterday. What's that `original_text` and how is it different from `OriginalText`?

Reply:
I have no memory of adding that in, but I presume I must have. Anyway, I've removed it and all tests still pass.